Solution1/BusinessLayer/Extension/WeatherDtoExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BusinessLayer.DTOs;
+
+namespace BusinessLayer.Extension
+{
+    public static class WeatherDtoExtensions
+    {
+        public static string PrintToString (this WeatherDTO weatherDTO)
```

Comment:
here is Get more suitable, cause you not printing, but returning string

Reply:
GetStringRepresentation for example

Reply:
+